pkg/identity/kubernetes_test.go

```diff
@@ -35,4 +35,6 @@ func TestGetKubernetesServiceIdentity(t *testing.T) {
 			assert.Equal(si, tc.expectedServiceIdentity)
 		})
 	}
+
+	assert.Equal(ServiceIdentity("blah").String(), "blah")
```

Comment:
Should we use something better rather than `blah` ?

Reply:
Yeah if we had something in test fixtures to use, that would be ideal

Reply:
This is fine really, its just testing the `String()` method on the object. I don't see why we need anything to go into the text fixtures for this.

Reply:
I was just wondering if there was an existing field in fixtures that would be convenient to reuse. 

We can also use “foo” or “bar” here. It would be better than “blah” :)